src/plugins/analysis/file_system_metadata/code/file_system_metadata.py

```diff
@@ -94,8 +94,8 @@ def _extract_metadata_from_file_system(self, file_object: FileObject):
             if output_file.is_file():
                 self._analyze_metadata_of_mounted_dir(json.loads(output_file.read_bytes()))
             else:
-                message = 'mount failed:\n{}'.format(output)
-                logging.warning('[{}] {}'.format(self.NAME, message))
```

Comment:
Templating `self.NAME` doesn't seem sensible in the case of these messages since there is no generic-ness to the messages and you could add explicit-y by replacing it.

Reply:
changed